common/metrics/registry.go

```diff
@@ -61,8 +64,13 @@ var (
 	// third-party metrics library in step 5 is used here and rendered in the response as comments like:
 	// # HELP <metric name> <metric description>.
 	globalRegistry registry
+
 	// errMetricAlreadyExists is returned by registry.buildCatalog when it finds two metrics with the same name.
 	errMetricAlreadyExists = errors.New("metric already exists")
+
+	// Read the env var TRACK_METRICS_LOCATION: if enabled, then store filename
+	// and line of metrics definition location.
+	trackMetricsLocation = environment.GetTrackMetricsLocation()
```

Comment:
how does sreee build the server? if it does its own build, can we do this with a build tag instead of an env var? if not.. can it do its own build instead?

Reply:
You can do something like define a function as a no-op in one file and with a real implementation in another, activated by a build tag.

I meant that.. does sreee use it as a dep, or use a binary and run it as a subprocess (to dump out some data or something).

Reply:
Ok, I pretty much redid the PR using build tags.